easybuild/easyconfigs/i/inferCNV/inferCNV-1.16.0-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
+
+name = 'inferCNV'
+version = '1.16.0'
+versionsuffix = '-R-%(rver)s'
+local_biocver = '3.15'
+
+homepage = 'https://github.com/broadinstitute/inferCNV/wiki'
+description = """InferCNV is used to explore tumor single cell RNA-Seq data to identify evidence
+ for somatic large-scale chromosomal copy number alterations, such as gains or
+ deletions of entire chromosomes or large segments of chromosomes."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('R', '4.2.1'),
+    ('R-bundle-Bioconductor', local_biocver, '-R-%(rver)s'),
+    ('rjags', '4-13', '-R-%(rver)s'),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/%s/bioc/src/contrib/' % local_biocver,
+        'https://bioconductor.org/packages/%s/bioc/src/contrib/Archive/%%(name)s' % local_biocver,
+        'https://bioconductor.org/packages/%s/data/annotation/src/contrib/' % local_biocver,
+        'https://bioconductor.org/packages/%s/data/experiment/src/contrib/' % local_biocver,
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+
+exts_defaultclass = 'RPackage'
+
+exts_list = [
+    ('phyclust', '0.1-33', {
+        'checksums': ['38786608193bcad6aeb5517f2d5f836c9773bdd09fc484b4bd40b06a71f5a413'],
+    }),
+    ('infercnv', version, {
+        'checksums': ['af7c53de0bc78d5a030fa695d35531f8bac6c8ea83a4b7097437490eff25d3a5'],
+    }),
+]
```

Comment:
There are two choices to fix the download failure:
1. Add the extra source download location for `inferCNV`, so that we know where to find it.
2. Revert this easyconfig, so that is just installs `inferCNV` and add the extra required package `phyclust` to the R easyconfig.